### PR TITLE
bcftools_consensus: add 'rename' parameter

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
This tool adds the ability to rename the consensus created by bcftools consensus. The common use case for this is taking a collection (list) of VCF (e.g. one VCF for each bacterial sample), and creating a consensus from each member of the list (combined with a reference sequence). In this case the new consensuses (consensi?) should not have the name of the reference sequence consensus but rather inherit a name from the VCF used to generate them. This works in an elegant way for list collections with items named after sample names.

Longer-term TODO: motivate for this option to be added to the bcftools code so that the use of awk is no longer necessary. For now the pipe to awk handles the renaming.

File: tools/bcftools/bcftools_consensus.xml [hunk 1]
```diff
@@ -38,8 +39,14 @@ ${section.iupac_codes}
 #end if
 
 ## Primary Input/Outputs
---output "$output_file"
+#if str($rename) == "no"
+    --output "$output_file"
```
Comment (bgruening) [inline tools/bcftools/bcftools_consensus.xml]: please use singe-quotes

Comment (pvanheus) [reply tools/bcftools/bcftools_consensus.xml]: done.

File: tools/bcftools/bcftools_consensus.xml [hunk 2]
```diff
@@ -25,21 +26,27 @@ bcftools @EXECUTABLE@
...
 #end if
 
 ## Primary Input/Outputs
---output "$output_file"
+#if str($rename) == "no"
+    --output '$output_file'
+#end if
 @INPUT_FILE@
+#if str($rename) == "yes":
+    #set basename=$input_file.name
```
Comment (mvdbeek) [inline tools/bcftools/bcftools_consensus.xml]:
```suggestion
    #set basename=$input_file.element_identifier
```

Comment (pvanheus) [reply tools/bcftools/bcftools_consensus.xml]: done. what exactly is the distinction here? I thought that historically 'element_identifier' was only for list elements?

**Pull request merged**